nextcord/http.py

```diff
@@ -453,6 +497,53 @@ async def static_login(self, token: str) -> user.User:
 
         return data
 
+    def set_client_secret(self, client_secret: str) -> None:
+        self.client_secret = client_secret
```

Comment:
Almost seems somewhat redundant?

Reply:
it was originally underscored, changed it when I realized the bot token wasn't private. I'll fix that maybe.